app/javascript/app/components/stories/stories-styles.scss

```diff
@@ -0,0 +1,114 @@
+@import '~styles/settings.scss';
+@import '~styles/layout.scss';
+
+.wrapper {
+  @include row();
+
+  flex-direction: column;
+  height: 1400px;
+
+  @media #{$tablet-portrait} {
+    height: 1080px;
+  }
+
+  @media #{$tablet-landscape} {
+    height: 800px;
+  }
+}
+
+.grid {
+  flex: 1;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+    'bg-story'
+    'small-story-1'
+    'small-story-2';
+  grid-gap: 20px;
+
+  > a {
+    grid-area: small-story-1;
+  }
+
+  & :last-child {
+    grid-area: small-story-2;
+  }
+
+  & :first-child {
+    grid-area: bg-story;
+  }
+
+  margin-bottom: 30px;
+
+  @media #{$tablet-landscape} {
+    grid-template-columns: 2fr 1fr;
+    grid-template-areas:
+      'bg-story small-story-1'
+      'bg-story small-story-2';
+  }
+}
+
+.story {
+  background-image: url('assets/story-2.png');
+  background-size: cover;
+  background-position: center;
+  height: 100%;
+  position: relative;
+
+  &:first-child { background-image: url('assets/story-1.png'); }
+  &:last-child { background-image: url('assets/story-3.png'); }
+}
+
+.storyDate {
+  color: $white;
+  font-size: $font-size-sm;
+  letter-spacing: 1.15px;
+  line-height: 17px;
+  padding: 30px;
+}
+
+.storyTitle {
+  font-family: $font-family-1;
+  color: $white;
+  font-size: $font-size-large;
+  font-weight: bold;
+  line-height: 30px;
+  max-width: 300px;
+  padding-left: 30px;
+}
+
+.storyDescription {
+  padding-left: 30px;
+  color: $white;
+  margin-top: 30px;
+  max-width: 400px;
+  line-height: 19px;
+}
+
+.logoContainer {
+  position: absolute;
+  bottom: 30px;
+  padding-left: 30px;
+  display: flex;
+  align-items: center;
+}
+
+.icon {
+  width: 30px;
+  height: 30px;
+}
+
+.text {
+  padding-left: 5px;
+  color: $white;
+  font-size: $font-size-sm;
+}
+
+.button {
+  background-color: $white;
+  color: $elephant;
+  max-width: 250px;
+  margin: 0 auto;
+  margin-bottom: 30px;
```

Comment:
Could we add some extra margin to the bottom? Maybe in the .wrapper

![image](https://user-images.githubusercontent.com/9701591/48347633-14dc0a00-e67f-11e8-8d3d-1487fcc9c597.png)